### DISCUSSION TITLE
Port ffmpegkit in maui and make it cross platform

### DISCUSSION THREAD

**Discussion by Barshan-Mandal** [CommunityToolkit/Maui#2111] [General]
Add support for FFMPEG in a cross platform manner .

implementation can be like this:

ffmpegkit ffmpegkit=new ffmpegkit();
ffmpegkit.convertvideo(parameters);
//to execute any command directly and return errors by try catch
ffmpegkit.executecommand(command);

**Comment by pictos** [comment]
I believe this is too huge to be part of our toolkit... What we already have on the MediaElement doesn't work for you?

**Reply by bijington** [reply]
I agree that it would be a lot of work that I'm not keen to take on. Perhaps you could start something and see how it goes?

**Comment by CliffAgius** [comment]
Closed as duplicate to #2127 which has more details.
